src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -246,10 +264,23 @@ export class NfsFormComponent extends CdForm implements OnInit {
               name: 'CEPH'
             })
           ]
+        }),
+        user_id: new UntypedFormControl('', {
+          validators: [
+            CdValidators.requiredIf({
+              name: 'RGW'
+            })
+          ]
         })
       }),
       subvolume_group: new UntypedFormControl(''),
       subvolume: new UntypedFormControl(''),
+      rgw_export_type: new UntypedFormControl(
+        null,
+        CdValidators.requiredIf({
+          'fsal.name': 'RGW'
+        })
+      ),
```

Comment:
Lets keep all rgw backend related fields together and cephfs together and common ones together.
```
//common
prop1:
prop2:
...
//cephfs


//rgw


}
```

Reply:
Done!